### PR TITLE
standaloneSkimCreator

### PR DESCRIPTION
**Pull request by nkuehnel**
updates to skim creator

### CODE REVIEW

File: extensions/matsim2silo/src/main/java/de/tum/bgu/msm/matsim/StandaloneSkimCreator.java [hunk 1]
```diff
@@ -28,37 +30,59 @@
...
         Network network = NetworkUtils.createNetwork();
         new MatsimNetworkReader(network).readFile(networkPath);
 
         Scenario scenario = ScenarioUtils.createScenario(config);
+        logger.info("Reading schedule...");
         new TransitScheduleReader(scenario).readFile(schedulePath);
         TransitSchedule schedule = scenario.getTransitSchedule();
 
+        logger.info("Reading shape file...");
         final Collection<SimpleFeature> features = ShapeFileReader.getAllFeatures(shapePath);
```
Comment (key2000) [inline extensions/matsim2silo/src/main/java/de/tum/bgu/msm/matsim/StandaloneSkimCreator.java:62]: I suppose error occurs when reading the shape file. here is the error message
https://gist.github.com/key2000/f8f55f5a412624fe72bab17967f4013d

**Review by key2000 — COMMENTED** [review]
I have made "VCS>update project", and reimport all maven project. Is it correct process to update the project?

File: extensions/matsim2silo/src/main/java/de/tum/bgu/msm/matsim/StandaloneSkimCreator.java (hunk 1 above)
Comment (nkuehnel) [reply extensions/matsim2silo/src/main/java/de/tum/bgu/msm/matsim/StandaloneSkimCreator.java:62]: Could you try again with the master version now? It runs on my machine :)

Comment (key2000) [reply extensions/matsim2silo/src/main/java/de/tum/bgu/msm/matsim/StandaloneSkimCreator.java:62]: Thank you for quick modification, but I have got same error in reading shape file.

https://gist.github.com/key2000/04cc80a8f5d593995b2d0bcacdda025c

**Pull request merged**